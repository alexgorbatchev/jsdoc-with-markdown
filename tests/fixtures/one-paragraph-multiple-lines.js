/**
This is a description.
And this is more of it.
*/
