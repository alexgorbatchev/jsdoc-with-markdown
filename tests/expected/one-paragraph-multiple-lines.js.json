[
  {
    "type": "paragraph",
    "value": "<p>This is a description. And this is more of it.</p>\n"
  }
]
